multichat/chat/consumers.py

```diff
@@ -1,6 +1,8 @@
 import json
 from channels import Channel
 from channels.auth import channel_session_user_from_http, channel_session_user
+
+from .settings import MSG_TYPE_LEAVE, MSG_TYPE_ENTER, NOTIFY_USERS_ON_ENTER_OR_LEAVE_ROOMS
```

Comment:
Why not just use django.conf.settings directly like most Django examples do for notify_users_on_enter? This seems like a needless indirection that will confuse people trying to read the code.